js/tc39/breaking_test_errors.json

```diff
@@ -2463,6 +2453,7 @@
   "test/language/import/escaped-as-import-specifier.js-strict:true": "test/language/import/escaped-as-import-specifier.js: unexpected error type (ReferenceError), expected (SyntaxError)",
   "test/language/import/escaped-as-namespace-import.js-strict:true": "test/language/import/escaped-as-namespace-import.js: unexpected error type (ReferenceError), expected (SyntaxError)",
   "test/language/import/escaped-from.js-strict:true": "test/language/import/escaped-from.js: unexpected error type (ReferenceError), expected (SyntaxError)",
+  "test/language/keywords/ident-ref-this.js-strict:true": "test/language/keywords/ident-ref-this.js: error is not an object (Test262: This statement should not be evaluated.)",
```

Comment:
`this` should not be usable as an identifier, this is the case for some other keywords as well